### PR TITLE
Better support phone number extraction

### PR DESCRIPTION
**Pull request by STayinloves**
- add tests in http://formvalidation.io/validators/phone/
- fix phone number extraction for cases like "010-6228-8559" and " 010-62288559"

### CODE REVIEW

File: Specs/Sequence/English/PhoneNumberModel.json [hunk 1]
```diff
@@ -883,6 +883,2692 @@
...
+    "Input": "My phone number is +44 (0) 161 1234567 ext 123.",
+    "NotSupportedByDesign": "python",
+    "NotSupported": "dotnet,javascript",
+    "Results": [
+      {
+        "Text": "+44 (0) 161 1234567 ext 123",
+        "TypeName": "phonenumber",
+        "Resolution": {
+          "value": "+44 (0) 161 1234567 ext 123"
+        }
```
Comment (yiylin) [inline Specs/Sequence/English/PhoneNumberModel.json]: do we need to include the invalid exmaples in the website which you share?

Comment (yiylin) [reply Specs/Sequence/English/PhoneNumberModel.json]: and you also can add "911", "110"

Comment (tellarin) [reply Specs/Sequence/English/PhoneNumberModel.json]: Not at this point, due to conflicts.

Comment (yiylin) [reply Specs/Sequence/English/PhoneNumberModel.json]: but can mark them as "notsupported"

**Pull request merged**